paymentsheet/src/main/java/com/stripe/android/customersheet/data/CustomerSheetDataResultKtx.kt

```diff
@@ -0,0 +1,15 @@
+package com.stripe.android.customersheet.data
+
+import com.stripe.android.customersheet.CustomerAdapter
+import com.stripe.android.customersheet.ExperimentalCustomerSheetApi
+
+@OptIn(ExperimentalCustomerSheetApi::class)
+internal fun <T> CustomerAdapter.Result<T>.toCustomerSheetDataResult(): CustomerSheetDataResult<T> {
```

Comment:
is there a reason we still need a custom result type for CustomerAdapter? I'm curious if we could just replace CustomerAdapterResult with CustomerSheetDataResult

Reply:
yep, makes sense. Thanks!